### PR TITLE
implement classes Car/CarWashStation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
+        cost_for_wash = round(
+            car.comfort_class
+            * (self.clean_power - car.clean_mark)
+            * self.average_rating
+            / self.distance_from_city_center,
+            1
+        )
+        return cost_for_wash
+
+    def wash_single_car(self, car: object) -> None:
```
Comment (ArsenPidhoretskyi) [inline app/main.py]: car is Car, not object!

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,55 @@
...
+                 average_rating: float,
+                 count_of_ratings: int
+                 ) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: object) -> float:
```
Comment (ArsenPidhoretskyi) [inline app/main.py]: car is Car, not object!

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,55 @@
...
+
+    def rate_service(self, new_rate: int) -> None:
+        fully_old_rate = self.average_rating * self.count_of_ratings
+        self.count_of_ratings += 1
+        self.average_rating = round(
+            (fully_old_rate + new_rate) / self.count_of_ratings,
+            1
+        )
+
+    def serve_cars(self, cars_box: list[object]) -> float:
```
Comment (ArsenPidhoretskyi) [inline app/main.py]: list of Car, not object
Rename cars_box to cars

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by vsmutok — APPROVED** [review]
great!

**Pull request closed without merging**